source/_components/weather.openweathermap.markdown

```diff
@@ -25,13 +25,15 @@ weather:
     api_key: YOUR_API_KEY
     latitude: 46.1234
     longitude: 7.1234
+    forecast: True
 ```
 
 Configuration variables:
 
 - **api_key** (*Required*): Your API key for http://openweathermap.org/.
 - **latitude** (*Optional*): Latitude of the location to display the weather. Defaults to the latitude in your your `configuration.yaml` file. 
 - **longitude** (*Optional*): Longitude of the location to display the weather. Defaults to the longitude in your `configuration.yaml` file.
+- **forecast** (*Optional*): Wether the plugin should retrieve forecast data. Defaults to False.
```

Comment:
The correct term would be "platform" but I would like to suggest to leave it out.